src/openforms/submissions/tests/test_admin.py

```diff
@@ -143,3 +146,72 @@ def test_change_view_displays_logs_if_not_avg(self):
 
         # avg log not visible
         self.assertNotContains(response, avg_log.get_message())
+
+
+class TestLogicEvaluated(VariablesTestMixin, WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        cls.submission = SubmissionFactory.from_data(
+            {
+                "firstname": "foo",
+                "birthdate": "2022-06-20",
+            },
+        )
+
+    def setUp(self):
+        super().setUp()
+        self.admin_user = UserFactory.create(
+            is_superuser=True, is_staff=True, app=self.app
+        )
+        self.anonymous_user = UserFactory.create(
+            is_superuser=False, is_staff=False, app=self.app
+        )
```

Comment:
we have a decorator to disable 2fa in `openforms.tests.utils` - better to just decorate the entire test case and then you can just set up your users in `setUpTestData` instead of here.